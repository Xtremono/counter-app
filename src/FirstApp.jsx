export const FirstApp = () => {
  return (
    <>
    <h1>Soy Tomas</h1>
    <p>Estoy tratando de conseguir mi primer trabajo en el mundo tech</p>
    </>
  );
};

// <></> A esto se le llama fragmento, es un agrupador de elementos html. Esto tambien lo conocemos como "Nodo Padre"
