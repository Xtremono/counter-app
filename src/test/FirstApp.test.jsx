
import { render } from "@testing-library/react"
import { FirstApp } from "../FirstApp"




describe('FirstApp', () => {

  test('Debe hacer match con el snapshot', () => {
      const title = "Hola, soy Tomas"
      const {container} = render(<FirstApp title={title}/>)

      expect(container).toMatchSnapshot();


  })

  test('debe de mostrar el titulo en un h1', () => {
    const title = "Hola, soy Tomas"
    const {container, getByText} = render(<FirstApp title={title}/>)
    expect(getByText(title)).toBeTruthy();

    const h1 = container.querySelector("h1");
    expect(h1.innerHTML).toBe(title);




   })
})
