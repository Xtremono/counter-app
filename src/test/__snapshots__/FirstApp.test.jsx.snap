// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FirstApp Debe hacer match con el snapshot 1`] = `
<div>
  <h1>
    Hola, soy Tomas
  </h1>
  <p />
  <p />
</div>
`;
